rasa/nlu/selectors/response_selector.py

```diff
@@ -78,16 +88,23 @@ class ResponseSelector(DIETClassifier):
         any_of(DENSE_FEATURE_NAMES[RESPONSE], SPARSE_FEATURE_NAMES[RESPONSE]),
     ]
 
-    # please make sure to update the docs when changing a default parameter
+    # default properties (DOC MARKER - don't remove)
```

Comment:
we don't include this in the docs anymore, you need to manually update the docs, e.g. this line can be removed